tests/unit/remoteworker/test_remote_worker.py

```diff
@@ -5,7 +5,9 @@
 
 from os.path import join
 
-from unittest import TestCase
+from rest_framework.test import APITestCase
+from participants.models import Participant
+from django.core.files.uploadedfile import SimpleUploadedFile
```

Comment:
@takitsuse `from django.core.files.uploadedfile import SimpleUploadedFile` should be at 8th line

Reply:
In this order?:
```python
from django...
from rest_...
from participants...
```